<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findDocument</class>
 <widget class="QDialog" name="findDocument">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="print">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>250</width>
     <height>350</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>25</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="overwrite">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-Overwrite</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>450</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="editInput">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>250</width>
     <height>350</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
